### PR TITLE
Foysal

### PR DESCRIPTION
**Pull request by foysal-mahmud-hasan**

### CODE REVIEW

File: src/components/modules/core/vendor/VendorForm.jsx [hunk 1]
```diff
@@ -49,6 +50,7 @@ function VendorForm(props) {
                 }
                 return null;
             },
+            customer_id : isNotEmpty()
```
Comment (Rashedul-Raju-3m) [inline src/components/modules/core/vendor/VendorForm.jsx:53]: ### why not empty ? it's not required field.

File: src/components/modules/core/vendor/VendorForm.jsx [hunk 2]
```diff
@@ -211,11 +214,7 @@ function VendorForm(props) {
                                             </Box>
                                             <Box mt={'xs'}>
                                                 <SelectForm
-                                                    tooltip={
```
Comment (Rashedul-Raju-3m) [inline src/components/modules/core/vendor/VendorForm.jsx:214]: why remove this ? if customer select & already exists , how to show the instant error message?

**Review by Rashedul-Raju-3m — CHANGES_REQUESTED** [review]
check this

File: src/components/modules/core/vendor/VendorForm.jsx (hunk 2 above)
Comment (Rashedul-Raju-3m) [reply src/components/modules/core/vendor/VendorForm.jsx:214]: may be you not up to date, because i fixed this issue today.

Comment (foysal-mahmud-hasan) [reply src/components/modules/core/vendor/VendorForm.jsx]: Bhai I rolled back changes in vendor form. kindly check .

**Pull request merged**